script/changeCanaryPackageVersion.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * This Script change all package.json fils under canary/ .
+ * It changes `@aws-amplify/ui-{framework}` version to `next`.
+ */
+
+import { globbyStream } from 'globby';
+import fs from 'fs';
+import { pathToFileURL } from 'url';
+
+for await (const path of globbyStream('canary/**/package.json')) {
+  const pkgJSON = JSON.parse(fs.readFileSync(path));
+  const { dependencies } = pkgJSON;
+  if (dependencies) {
+    let pkgToUpdate;
+    Object.keys(dependencies)
+      .filter((dependency) => dependency.includes('@aws-amplify'))
```

Comment:
Very very nit coming from me, what do you think about 
```suggestion
      .filter((dependency) => dependency.includes('@aws-amplify/ui'))
```
so that it filters out any non-ui Amplify packages like `@aws-amplify/auth`? We do not ever use packages directly (they're installed indirectly through `aws-amplify` package), but just for the sake of explicitness.